src/pyfmi/fmil_import.pxd

```diff
@@ -1017,5 +1017,3 @@ cdef extern from 'fmilib.h':
 
```

Comment:
Should there not be an extra method referenced here? (the import_and_get...)

Reply:
I referenced the function in fmi.pxd, does it make more sense to have it in fmil_import.pxd?

Reply:
Everything that reference functions from FMIL should go into fmil_import.pxd

Reply:
Oh I see! But the function import_and_get is not a new function from FMIL, it is defined in fmi.pyx, and it uses the same FMIL function as with the old behavior so there is no new FMIL function in this implementation.

Reply:
Ok!